RELEASE_NOTES.md

```diff
@@ -1,5 +1,47 @@
 # Release Notes
 
+## Changes in Splinter 0.5.5
+
+### Highlights
+
+* The `admin/circuit` route now returns a list of nodes in the member list
+  instead of just the node id if the `SplinterProtocolVersion` is set to 2.
+  This is a breaking change from previous versions.
+
+* Stabilized authorization-handler-rbac feature.  This enables authorizing REST
```

Comment:
Maybe code-quote the feature here, for consistency with the other features in this change.

Reply:
Updated instances of this feature that did not have code-quotes